eZ/Bundle/EzPublishCoreBundle/EventListener/SessionInitByPostListener.php

```diff
@@ -27,7 +27,7 @@ public static function getSubscribedEvents()
     public function onSiteAccessMatch(PostSiteAccessMatchEvent $event)
     {
         $request = $event->getRequest();
-        $session = $request->getSession();
+        $request->hasSession() ? $session = $request->getSession() : $session = false;
```

Comment:
Using ternary logic operator like this makes the code harder to read.

```suggestion
        $session = null;
        if ($request->hasSession()) {
            $session = $request->getSession();
        }
```

Additionally, `Session|null` is better than `Session|false`, since it does not mix different data types.

Reply:
I agree, but I though you preferred the termary logic operator